extensions/shared/src/main/java/app/revanced/extension/music/patches/misc/requests/PlaylistRequest.kt

```diff
@@ -98,7 +98,7 @@ class PlaylistRequest private constructor(
             Objects.requireNonNull(videoId)
             synchronized(cache) {
                 val now = System.currentTimeMillis()
-                cache.values.removeIf { request: PlaylistRequest ->
+                cache.values.removeAll { request ->
```

Comment:
Not that important, but for consistency with the [previous PR](https://github.com/inotia00/revanced-integrations/pull/57), I'll use `isSDKAbove(24)` here too.